backend/src/v2/component/launcher_v2.go

```
@@ -523,6 +541,47 @@ func execute(
 	return getExecutorOutputFile(executorInput.GetOutputs().GetOutputFile())
 }
 
+// Append custom CA to system CA bundle and store in temp file
+func compileTempCABundle(customCAMountPath string) string {
+	// Possible certificate files; stop after finding one.
+	var systemCAs = []string{
+		"/etc/ssl/certs/ca-certificates.crt",
+		"/etc/pki/tls/certs/ca-bundle.crt",
+		"/etc/ssl/ca-bundle.pem",
+		"/etc/pki/tls/cacert.pem",
+		"/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem",
+		"/etc/ssl/cert.pem",
+	}
+
+	tmpCaBundle, err := os.CreateTemp("", "ca-bundle-*.crt")
+	if err != nil {
+		glog.Errorf("Error creating temp file: %s", err)
+	}
+	for _, file := range systemCAs {
+		systemCaBundle, err := os.ReadFile(file)
+		if err == nil {
+			if _, err = tmpCaBundle.Write(systemCaBundle); err != nil {
+				glog.Errorf("Error writing to temp ca-bundle file: %s", err)
```

Should this not return an error instead of logging and continuing?

---

Updated